### DISCUSSION TITLE
Whisper models appear to be missing in new SubtitleEdit install, DL's of models destroy existing models

### DISCUSSION THREAD

**Discussion by LastJasonNext** [SubtitleEdit/subtitleedit#7079] [Q&A]
Has anyone else experienced a fail when Downloading Whisper models via the SubtitleEdit Whisper dialog? 

Warning: if you experience what I have when attempting to check for updated models via SubTitleEdit Whisper dialog, any selected model for download will be replaced with a 1K empty file. I have looked in vain for a replacement (base.en.bin), and am now becoming familiar with Python in an attempt to garner them in .bin form from Huggingface.

I saved the 3 models that hadn't been destroyed (by attempting a fresh Download in the Whisper dialog), did a complete uninstall then installed SubtitleEdit 3.6.13. After installation, the Whisper dialog asked to DL ffmpeg and cpp run files, which appear successful. No model options are preloaded into the model dropdown menu, and a Download selection of a model results in the loading of a 1K empty model file. So a complete reinstall of SubtitleEdit 3.6.13 will not resurrect the destroyed base.en.bin.

As I'm looking for a base.en.bin replacement, perhaps someone might have a link to one?

As another avenue while searching for replacement(s), I asked CharGPT 4.0 for a downloadable address for the .bin models. It was happy to give me one at Huggingface, however that folder appears to be empty. We discussed the difference between .pt models (reformed for python usage) and the original .bin models, so I'm fairly sure GPT 4.0 was accurate with the link. Perhaps someone could try that as well (I used Bing's).

I did drop a line to Nikolaj asking for guidance here, but haven't heard back. If its not a "just me" situation, he may be understandably busy.

-J

Any advice on cloning from GitHub / Huggingface and downloading models would be appreciated, mostly I'd jsut like to know if its a "just me" situation for whatever reason.

Thanks to all.

**Comment by niksedk** [comment]
The links has changed: https://github.com/SubtitleEdit/subtitleedit/commit/cf429947956e60d9f3236b9c2153e8bdcea503b1

Fixed in latest beta: https://github.com/SubtitleEdit/subtitleedit/releases/download/3.6.13/SubtitleEditBeta.zip

Duplicate of:
https://github.com/SubtitleEdit/subtitleedit/issues/7060
https://github.com/SubtitleEdit/subtitleedit/issues/7074
